curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b4ae6f9d0dc04a3705dcd.md

```diff
@@ -0,0 +1,108 @@
+---
+id: 646b4ae6f9d0dc04a3705dcd
+title: Step 17
+challengeType: 0
+dashedName: step-17
+---
+
+# --description--
+
+You will create a `option` HTML element with the `value` attribute set to `dd-mm-yyyy` and set the Text inside to `Day, Month, Year`
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed